python/Ice/bidir/server/main.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# Copyright (c) ZeroC, Inc.
+
+import Ice
+import asyncio
+from wakeUpService import BidirWakeUpService
```

Comment:
Maybe the convention should be like in the greeter demo: servant class goes into a file named after this class.

So Chatbot => chabot.py and later referenced as chatbot.Chatbot

BidirWakeUpService => bidir_wake_up_service.py and later referenced as bidir_wake_up_service.BidirWakeUpService

Or can you think of a better model?

Reply:
or maybe just`greeter.py` `bidir.py` and import it as:

`from greeter import Chatbot`, `from bidir import BidirWakeUpService`

Reply:
I definitely prefer chatbot.py over greeter.py.

Reply:
What about bidir.py?